frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/vm-settings-tab/url-source.tsx

```diff
@@ -34,6 +36,17 @@ export const URLSource: React.FC<URLSourceProps> = React.memo(
             }
           />
         </FormField>
+        <div className="pf-c-form__helper-text" aria-live="polite">
+          For example, you can obtain the download link for {isUpstream ? 'Fedora' : 'RHEL'} cloud
```

Comment:
can we change this to:
"Visit the Fedora cloud image[link] list and copy a url to paste into the url field here"

Reply:
@glekner ^^